tv2.html

```diff
@@ -0,0 +1,283 @@
+---
+layout: default
+permalink: /tv2
+---
+<section class="bg-tan-light soft-sides soft-top">
+  <div class="soft-sides soft-top">
+    <div class="container soft-top hard-sides">
+      <div class="col-md-8">
+        <div class="latest-message">
+          <div class="latest-message-background soft-half-bottom push-bottom">
+            <div class="embed-container">
+              <a href="#item" data-automation-id="message-video">
+                <div class="media-video">
+                  <div class="card-video-player">
+                    <div class="embed-responsive embed-responsive-16by9" id="js-media-video" player-id="player"
+                      video-id="9vmwUS1E-Lc" data-video-player>
+                      <div id="player"></div>
+                    </div>
+                  </div>
+                </div>
+              </a>
+            </div>
+            <h2 class="latest-message-headline font-family-condensed-extra text-uppercase flush-top text-gray-dark">
+              IS GOD GOOD? | THE GOOD GOD JOURNEY WEEK 1
```

Comment:
I find it is a best practice to make titles title case so you would have "Is God Good? | The Good God Journey Week 1" and then use CSS `text-transform: uppercase;` to force it to uppercase

Reply:
@dholloran-ample Yes, good point. I was planning to put this in the repo but wanted to get something over to AG and Sarah for review while I was still working on some stuff with the subscribe modal. So the page in Contentful was really just intended to be something for them to preview quickly while I was finishing up. I hate permanently putting things in Contentful unless I'm explicitly asked to. This is why I was using <style> tags as well just to make it "work" until I was ready to commit everything to the repo. Hope that helps explain some of the mess - sorry you had to weed through that.